src/cargo-fmt/main.rs

```diff
@@ -334,41 +318,41 @@ fn add_targets(target_paths: &[cargo_metadata::Target], targets: &mut HashSet<Ta
 }
 
 fn run_rustfmt(
-    files: &[PathBuf],
+    targets: &HashSet<Target>,
     fmt_args: &[String],
     verbosity: Verbosity,
-) -> Result<ExitStatus, io::Error> {
-    let stdout = if verbosity == Verbosity::Quiet {
-        std::process::Stdio::null()
-    } else {
-        std::process::Stdio::inherit()
-    };
```

Comment:
This is something I don't know how to handle as I cannot inherit it and use it on multiple calls for `Command` as it is consumed. Does anyone have a hint on how to deal with this?